Azure/sonic-buildimage
# [Inventec]: Fixed D7054 missing front-panel port Interfaces

[device/inventec/x86_64-inventec_d7054q28b-r0/INVENTEC-D7054Q28B-S48-Q6] : Fixed D7054 missing to show front-panel port Interfaces in SONiC.

* Removed extra row "# 48x25G + 6x100G".
* Replaced "tab" as "space" and fixed alignment.

Signed-off-by: Lo.Wilson 羅仕佑 IHS <lo.wilson@inventec.com>


- How to verify it
Tested with "ifconifg -a" command.

- Description for the changelog
Start to see this issue since Project buildimage-brcm-all, Build #369 (Oct 24, 2017 9:20:58 AM).
